src/utils/web.ts

```diff
@@ -33,15 +34,15 @@ export async function fetchJson(connection: string | ConnectionInfo, json?: stri
             });
             if (!response.ok) {
                 if (response.status === 503) {
-                    console.warn(`Retrying HTTP request for ${url} as it's not available now`);
+                    getLogger().debug(`Retrying HTTP request for ${url} as it's not available now`);
```

Comment:
Looks like a `warning`.

Reply:
I chose debug for this one because it automatically retries. Most developers probably won't care.